src/assets/scripts/client/aircraft/AircraftCommander.js

```diff
@@ -592,19 +575,23 @@ export default class AircraftCommander {
             taxiDestination = airport.departureRunwayModel.name;
         }
 
-        const runway = AirportController.airport_get().getRunway(taxiDestination.toUpperCase());
+        const runwayModel = AirportController.airport_get().getRunway(taxiDestination.toUpperCase());
+
+        if (!runwayModel) {
+            return [false, `unable to find Runway ${taxiDestination.toUpperCase()} on our charts`];
+        }
 
-        if (!runway) {
-            return [false, `no runway ${taxiDestination.toUpperCase()}`];
+        if (!aircraft.fms.isRunwayModelValidForSid(runwayModel)) {
+            aircraft.pilot.cancelDepartureClearance(aircraft);
```

Comment:
does `.cancelDepartureClearance()` actually need the `aircraft` param?  seems a little weird, since it's the same instance calling that method.

Reply:
classic case of kill-the-pilot. When we get to removing it, it would simply be `aircraft.cancelDepartureClearance()`. Doing it this ugly way for now to keep with the convention.

Reply:
#worksforme